packages/graphback-core/src/runtime/subscriptionFilter.ts

```diff
@@ -0,0 +1,172 @@
+    const parsedFieldValue = convertType(fieldValue, typeof filterValue);
+    const parsedFilterValue = convertType(filterValue, typeof parsedFieldValue)
+
+    return parsedFilterValue !== parsedFieldValue;
+  },
+  gt: (filterValue: InputType) => (fieldValue: InputType) => {
+    const parsedFieldValue = convertType(fieldValue, typeof filterValue);
+    const parsedFilterValue = convertType(filterValue, typeof parsedFieldValue)
+
+    return parsedFieldValue > parsedFilterValue;
+  },
+  ge: (filterValue: InputType) => (fieldValue: InputType) => {
+    const parsedFieldValue = convertType(fieldValue, typeof filterValue);
+    const parsedFilterValue = convertType(filterValue, typeof parsedFieldValue)
+
+    return parsedFieldValue >= parsedFilterValue;
+  },
+  le: (filterValue: InputType) => (fieldValue: InputType) => {
+    const parsedFieldValue = convertType(fieldValue, typeof filterValue);
+    const parsedFilterValue = convertType(filterValue, typeof parsedFieldValue)
+
+    return parsedFieldValue <= parsedFilterValue;
+  },
+  lt: (filterValue: InputType) => (fieldValue: InputType) => {
+    const parsedFieldValue = convertType(fieldValue, typeof filterValue);
+    const parsedFilterValue = convertType(filterValue, typeof parsedFieldValue)
+
+    return parsedFieldValue < parsedFilterValue;
+  },
+  in: (filterValue: InputType[]) => (fieldValue: InputType) => {
+    return filterValue.map((f: InputType) => f?.toString()).includes(fieldValue?.toString());
+  },
+  between: ([fromVal, toVal]: InputType[]) => (fieldValue: InputType) => {
+    if (isDateObject(fieldValue)) {
+      const fieldValDate = convertType(fieldValue, typeof fieldValue)
+      const fromValDate = convertType(fromVal, typeof fieldValue)
+      const toValDate = convertType(toVal, typeof fieldValue)
+
+      return fieldValDate >= fromValDate && fieldValDate <= toValDate;
+    }
+
+    const parsedFieldValue = Number(fieldValue)
+
+    return parsedFieldValue >= Number(fromVal) && parsedFieldValue <= Number(toVal);
+  },
+  contains: (filterValue: InputType = '') => (fieldValue: InputType = '') => {
+    return fieldValue?.toString().includes(filterValue?.toString());
+  },
+  startsWith: (filterValue: string = '') => (fieldValue: string = '') => {
+    return fieldValue?.toString().startsWith(filterValue?.toString());
+  },
+  endsWith: (filterValue: string = '') => (fieldValue: string = '') => {
+    return fieldValue?.toString().endsWith(filterValue?.toString());
+  }
+}
+
+/**
+ * Dynamically creates a subscription filter predicate using the filter object values
+ * 
+ * @param {QueryFilter} filter - subscription filter input object
+ */
+export function createSubscriptionFilterPredicate<T = any>(filter: QueryFilter): (input: Partial<T>) => boolean {
+  const andFilter = filter.and;
+  const orFilter = filter.or;
+  const notFilter = filter.not;
+
+  const filterFields = Object.keys(filter).filter((key: string) => !['and', 'or', 'not'].includes(key));
```

Comment:
not needed we can skip "boolean operators" in the loop, https://github.com/aerogear/graphback/pull/1835/files#diff-5cab2776f685ff7a7ce3e24f71aeea51R99